frontend/src/reducers/degreeSlice.js

```diff
@@ -3,8 +3,7 @@ import { createSlice } from "@reduxjs/toolkit";
 const initialState = {
   programCode: "",
   programName: "",
-  majors: [],
-  minors: [],
+  specs: [],
```

Comment:
this is the localstorage thing i messed up - i think its a trivial thing to fix using a migration though, right?

Reply:
should be easy to create a migration function between versions. Though you may want to test it beforehand if it migrates correctly